lib/logflare/backends/adaptor/postgres_adaptor/shared_repo.ex

```diff
@@ -62,6 +74,12 @@ defmodule Logflare.Backends.Adaptor.PostgresAdaptor.SharedRepo do
 
     try do
       func.()
+    rescue
+      _e in DBConnection.ConnectionError ->
+        {:error, :cannot_connect}
```

Comment:
```suggestion
      DBConnection.ConnectionError ->
        {:error, :cannot_connect}
```

Technically that `_e in` thingy is not needed. I do not have strong opinion which one is more readable though.